src/GlobalVars.res

```diff
@@ -7,7 +7,8 @@
 @val external logEndpoint: string = "logEndpoint"
 @val external sentryDSN: string = "sentryDSN"
 @val external sentryScriptUrl: string = "sentryScriptUrl"
+@val external enableLogging: bool = "enableLogging"
```

Comment:
DefinePlugin of webpack will give you either a code fragment or a string. It will not give a boolean, so it is better to consume it as a string and decode it to a boolean. Another comment in the webpack.common.js file explains it further